src/bentoml/utils.py

```diff
@@ -0,0 +1,3 @@
+"""Reusable export of some internal utilities that are useful to used in adjacent projects."""
+from ._internal.utils import LazyLoader as LazyLoader
+from ._internal.utils import bentoml_cattr as bentoml_cattr
```

Comment:
This is to prevent the unused import warning from the linter (F401), and ruff --fix will fix F401 by default.

Maybe we can enable unfixable for ruff in pyproject.toml .
```
unfixable = ["F401"]
```

Reply:
Oh I didn't know the as-import suppresses the F401 warning. I usually add a `__all__` to hold their names.

Reply:
I'm also in favor of adding these to `__all__` if they're meant to be imported.

I'm not sure why we're exposing these, though?

Reply:
I think having a reusable utilities is kinda nice. Although for the most part on other adjacent libraries we can just do `from bentoml._internal.utils import ...`.